awx/main/credential_plugins/hashivault.py

```diff
@@ -34,10 +34,15 @@
         'help_text': _('The CA certificate used to verify the SSL certificate of the Vault server')
     }],
     'metadata': [{
+        'id': 'secret_backend',
+        'label': _('Name of Secret Backend'),
+        'type': 'string',
+        'help_text': _('The name of the kv secret backend (if left empty, the first part of the secret path will be used).')
```

Comment:
Nitpick since we're here:

"The first part of"

"The first path segment" of

https://tools.ietf.org/html/rfc3986
>  A path consists of a sequence of path segments separated by a slash ("/") character.

Reply:
good idea - done.